src/docs/deployment/cd.md

```diff
@@ -1,48 +1,50 @@
 ---
-title: Continuous Delivery using fastlane with Flutter
-description: How to use fastlane to automate continuous building and releasing of your Flutter app.
+title: Continuous Delivery with Flutter
```

Comment:
Delivery => delivery
(We use sentence case for titles, though I see we missed this title before...)